apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html

```diff
@@ -1004,27 +1004,27 @@
                                 </div>
 
                                 <div class="form-group col-xs-4">
-                                    <input type="text" name="shippedVia" [(ngModel)]="invFormData.templateDetails.other.shippedVia" class="form-control" placeholder="Ship Via" />
+                                    <input type="text" name="shippedVia" [(ngModel)]="invFormData.templateDetails.other.shippedVia" class="form-control" [placeholder]="templatePlaceholder?.shippedViaLabel || 'Ship Via'" />
                                 </div>
 
                                 <div class="form-group col-xs-4">
-                                    <input type="text" name="trackingNumber" [(ngModel)]="invFormData.templateDetails.other.trackingNumber" class="form-control" placeholder="Tracking No." />
+                                    <input type="text" name="trackingNumber" [(ngModel)]="invFormData.templateDetails.other.trackingNumber" class="form-control" [placeholder]="templatePlaceholder?.trackingNumber || 'Tracking No.'" />
                                 </div>
 
                             </div>
 
                             <div class="row">
 
                                 <div class="form-group col-xs-4">
-                                    <input type="text" name="customField1" [(ngModel)]="invFormData.templateDetails.other.customField1" class="form-control" placeholder="Field 1" />
+                                    <input type="text" name="customField1" [(ngModel)]="invFormData.templateDetails.other.customField1" class="form-control" [placeholder]="templatePlaceholder?.customField1Label || 'Field 1'" />
                                 </div>
 
                                 <div class="form-group col-xs-4">
-                                    <input type="text" name="customField2" [(ngModel)]="invFormData.templateDetails.other.customField2" class="form-control" placeholder="Field 2" />
+                                    <input type="text" name="customField2" [(ngModel)]="invFormData.templateDetails.other.customField2" class="form-control" [placeholder]="templatePlaceholder?.customField2Label || 'Field 2'" />
                                 </div>
 
                                 <div class="form-group col-xs-4">
-                                    <input type="text" name="customField3" [(ngModel)]="invFormData.templateDetails.other.customField3" class="form-control" placeholder="Field 3" />
+                                    <input type="text" name="customField3" [(ngModel)]="invFormData.templateDetails.other.customField3" class="form-control" [placeholder]="templatePlaceholder?.customField2Label || 'Field 3'" />
```

Comment:
It should be customField3Label here i guess?

Reply:
Done